### PR TITLE
Python client content and TOC reorg

### PR DESCRIPTION
**Pull request by boltzmann-brain**
Re-orged TOC to hvae harness/probe/detector section.
Harness topic is still incomplete because I need to paste in tables for every custom harness.

Replaced outdated python client info with latest info.

Hid orphaned topics like CLI, reporting, custom tests. These can be restored later.

### CODE REVIEW

File: source/components/harnesses.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+client.score.evaluations.create(
+    model_hub="openai",
+    model_name="gpt-3.5-turbo",
+    model_params={"temperature": 0},
+    harnesses=["ethics", "toxicity"],
+    harness_params={"sample_size": 1, "is_lite": False}
+)
+```
+
+## Other pre-configured harnesses
```
Comment (boltzmann-brain) [inline source/components/harnesses.md:36]: this part is not complete yet. I need to set aside a couple of hours to complete it from the spreadsheet.

File: source/components/detectors.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Detectors
+
+The following table summarizes all detectors used in Vijil. The module name is how the detector will appear in some logs.
+
+| Detector                             | Module name                                                      | Description                                                                                                                                                                                                            |
```
Comment (shubhobm) [inline source/components/detectors.md:5]: i don't think we're surfacing the module names to the end user. so we can delete that column?

File: source/getting-started.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ Let's begin by preparing your environment.
 We first install libraries necessary to run Vijil services.
 
 ```bash
-pip install autoredteam vijil-cli
+pip install vijil-python
```
Comment (shubhobm) [inline source/getting-started.md:10]: `pip install vijil`

File: source/getting-started.md
Comment (shubhobm) [inline source/getting-started.md:1]: Suggest adding all providers from the [website](https://www.vijil.ai/) carousel

File: source/python-sdk/structure/perturbations.md
Comment (shubhobm) [inline source/python-sdk/structure/perturbations.md:1]: i think we can delete perturbations for now

File: source/cli/commands.md
Comment (shubhobm) [inline source/cli/commands.md:1]: we can delete cli.md for now

**Review by shubhobm — CHANGES_REQUESTED** [review]

File: source/components/detectors.md (hunk 2 above)
Comment (boltzmann-brain) [reply source/components/detectors.md:5]: will they also be purged from the logs?

Comment (shubhobm) [reply source/components/detectors.md:5]: ah, good point. no i think we should keep them in the logs.

where i was coming from is the preference of keeping the documentation itself clean of any IDs used in the backend. maybe a page linked from here that lists the name to ID mapping of all entities?

Comment (boltzmann-brain) [reply source/components/detectors.md]: removed them; will add other page in diff PR

File: source/cli/commands.md
Comment (boltzmann-brain) [reply source/cli/commands.md:1]: done

File: source/getting-started.md
Comment (boltzmann-brain) [reply source/getting-started.md:1]: done

File: source/getting-started.md (hunk 3 above)
Comment (boltzmann-brain) [reply source/getting-started.md]: fixed

File: source/python-sdk/structure/perturbations.md
Comment (boltzmann-brain) [reply source/python-sdk/structure/perturbations.md:1]: done

**Comment by boltzmann-brain** [comment]
@shubhobm Addressed all your comments

**Review by shubhobm — APPROVED** [review]
lgtm

**Pull request merged**